hypershift-operator/controllers/nodepool/metrics/metrics.go

```diff
@@ -24,6 +24,12 @@ var (
 		Name: NodePoolAvailableReplicasMetricName,
 	}, labelNames)
 
+	NodePoolInfoMetricName = "hypershift_nodepools_info"
+	nodePoolInfo           = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Help: "Extra information associated to a given NodePool",
+		Name: NodePoolInfoMetricName,
+	}, append(labelNames, "_id", "ec2_instance_type"))
+
```

Comment:
**Question:** would it make sense to merge this with `hypershift_nodepool_replicas_available`?  Meaning just adding the `ec2_instance_type` and `_id` labels to that arleady existing metric, which is supposed to provide information about the nodepool?

Reply:
It is usually a good practice to factorise the labels which are not used to identify a time series in a dedicated `_info` metric:
- This allows to explicit the semantic of a label (i.e. whether this is a label used to identify a time series or not)
- Also this lower the memory pressure on Prometheus / on the TSDB (this effect is however quite minimal here as there is only one time series per hosted cluster for each of those "nodepools" metric)